examples/get_auth_token.py

```diff
@@ -0,0 +1,68 @@
+"""
+This example demonstrates how to get an authorization token for a user and then use it to make a request to the Google API on behalf of the user.
+"""
+
+from arcadepy import Arcade
+from arcadepy.types.auth_authorize_params import (
+    AuthRequirement,
+    AuthRequirementOauth2,
+)
+from google.oauth2.credentials import Credentials  # pip install google-auth
+from googleapiclient.discovery import build  # pip install google-api-python-client
+
+
+def get_auth_token(client: Arcade, user_id: str) -> str:
+    """Get an authorization token for a user.
+
+    In this example, we are
+        1. Starting the authorization process for the Gmail Readonly scope
+        2. Waiting for the user to authorize the scope
+        3. Getting the authorization token
+        4. Using the authorization token to make a request to the Google API on behalf of the user
+    """
+    # Start the authorization process
+    auth_response = client.auth.authorize(
```

Comment:
use `client.auth.start` instead, it has a much nicer interface to do the same thing.

Reply:
addressed